src/Gui/Action.cpp

```diff
@@ -826,8 +826,8 @@ void RecentFilesAction::restore()
         _group->addAction(QLatin1String(""))->setVisible(false);
     std::vector<std::string> MRU = hGrp->GetASCIIs("MRU");
     QStringList files;
-    for (std::vector<std::string>::iterator it = MRU.begin(); it!=MRU.end();++it)
-        files.append(QString::fromUtf8(it->c_str()));
+    for (auto & it : MRU)

```

Comment:
Is there a reason not to use "const auto &" here? (I recognize the old code was also non-const, but as long as we're improving the line anyway...)